R/qsys_ssh.r

```diff
@@ -55,7 +55,12 @@ SSH = R6::R6Class("SSH",
         fill_options = function(...) {
             args = list(...)
             args$local_port = sub(".*:", "", private$addr)
-            args$ssh.hpc_fwd_port = getOption("clustermq.ssh.hpc_fwd_port", sample(50000:55000, 1))
+            args$ssh.hpc_fwd_port = getOption(
+                "clustermq.ssh.hpc_fwd_port", 
+                sample(
+                    getOption("clustermq.ssh.hpc_fwd_port.range",50000:55000),1
```

Comment:
Why do you want to provide an ssh range option here in addition to a ssh port option? Wouldn't it be better to accept both a range and a single port in the port option?

In the latter case: move the `sample` on L58 outside the `getOption`

Reply:
That's a great idea - i initially just did not want to interfere with the current parameters too much but I now have extended `clustermq.ssh.hpc_fwd_port` to also accept a range. 

Unfortunately moving the `sample` outside of the `getOption` does not work too well if `clustermq.ssh.hpc_fwd_port` is set to a single value. But the solution in the latest commit definitely works both with single ports and port ranges.